test/homework_test/01_variables_test/01_variables_tests.cpp

```diff
@@ -12,3 +12,13 @@ TEST_CASE("Verify sum_numbers function")
 	REQUIRE(add_numbers(10, 10) == 20);
 }
 
+TEST_CASE("Verify get_sales_tax_amount function")
+{
+	REQUIRE(get_sales_tax_amount(10) == 0.675);
+	REQUIRE(get_sales_tax_amount(20) == 1.35);
+}
+
+TEST_CASE("Verify get_tip_amount function")
+{
+	REQUIRE(get_tip_amount(20, 0.20) == 4);
```

Comment:
Missing assertion to test with 15% tip.
REQUIRE(get_tip_amount(20, 0.15) == 3);